File: 1-Iniciante/Ex1005.java
import java.util.Scanner;

public class Ex1005 { //para ser aceito no URI, mude o nome da classe para Main
   public static void main(String args[]){
        //variaveis
        Double a, b, peso1 = 3.5, peso2= 7.5, media;
        Scanner entrada = new Scanner(System.in);

        //entrada
        a = entrada.nextDouble();
        b = entrada.nextDouble();

        //processamento
        media = (a * peso1 + b * peso2) / (peso1 + peso2);

        //saida
        System.out.printf("MEDIA = %.5f\n", media);
        entrada.close();
   } 
}

File: 1-Iniciante/Ex1005.kt
import java.util.Scanner

fun main(args: Array<String>) {
    val scanner = Scanner(System.`in`)
    val nota1 = scanner.nextDouble()
    val nota2 = scanner.nextDouble()

    val peso1 = 3.5
    val peso2 = 7.5

    val media = ((nota1 * peso1) + (nota2 * peso2)) / (peso1 + peso2)
    println("MEDIA = ${String.format("%.5f", media)}")
}

